### DISCUSSION TITLE
can you change the bot command or add a command?

### DISCUSSION THREAD

**Discussion by fuhmi69** [jagrosh/MusicBot#777] [Questions] [answered]
for example
-play to -p or -forceskip to -fs

so it is much faster to use the bot

**Comment by jagrosh** [comment, marked as answer]
Yes, you can specify aliases in the `aliases` section of the [config](https://github.com/jagrosh/MusicBot/wiki/Example-Config).

**Reply by alexysarts** [reply]
Yeah it's not working for me for some reason. The new aliases don't work.
